webpack/JobInvocationDetail/JobInvocationSystemStatusChart.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ChartDonut, ChartLabel, ChartLegend } from '@patternfly/react-charts';
+import {
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  FlexItem,
+  Text,
+} from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+import './JobInvocationDetail.scss';
+
+const JobInvocationSystemStatusChart = ({
+  data,
+  isAlreadyStarted,
+  formattedStartDate,
+}) => {
+  const {
+    succeeded,
+    failed,
+    pending,
+    cancelled,
+    total,
+    total_hosts: totalHosts,
+  } = data;
+  const chartData = [
+    { title: __('Succeeded:'), count: succeeded, color: '#3E8635' },
+    { title: __('Failed:'), count: failed, color: '#C9190B' },
+    { title: __('In Progress:'), count: pending, color: '#2B9AF3' },
+    { title: __('Canceled:'), count: cancelled, color: '#6A6E73' },
+  ];
+  const chartDonutTitle = () => {
+    if (total > 0) return `${succeeded.toString()}/${total}`;
+    if (totalHosts > 0) return `0/${totalHosts}`;
+    return '0';
```

Comment:
What's the difference between `total` and `totalHosts`?

Reply:
The `total` represents the combined count of succeeded, failed, pending, and canceled, whereas `totalHosts` also includes those currently scheduled in the future.